SIL.Windows.Forms.DblBundle/SelectProjectDlgBase.cs

```diff
@@ -8,30 +8,37 @@ namespace SIL.Windows.Forms.DblBundle
 {
 	public abstract class SelectProjectDlgBase : IDisposable
 	{
-		private readonly OpenFileDialog m_fileDialog;
+		private OpenFileDialog m_fileDialog;
+		private string m_defaultDir;
+		private bool m_allowProjectFiles;
 
 		protected abstract string DefaultBundleDirectory { get; set; }
 		protected abstract string ProjectFileExtension { get; }
 		protected abstract string Title { get; }
 		protected abstract string ProductName { get; }
 
-		protected SelectProjectDlgBase(bool allowProjectFiles = true, string defaultFile = null)
+		protected SelectProjectDlgBase(bool allowProjectFiles, string defaultFile = null)
```

Comment:
We shouldn't change the signature of the c'tor!

```suggestion
		protected SelectProjectDlgBase(bool allowProjectFiles = true, string defaultFile = null)
```